include/cbl++/Replicator.hh

```diff
@@ -17,7 +17,11 @@
 //
 
 #pragma once
+
 #include "CBLReplicator.h"
+
+#include "Document.hh"
```

Comment:
Was this fixing a particular issue?

Reply:
The issue is part of making the project to compile on Visual Studio after modifying the CBL.def: When Replicator.hh is referenced from a different project, on this line `using ReplicationFilter = std::function<bool(Document, bool isDeleted)>;`, Document is an undeclared identifier